risingstars.js.org

michaelrambeau opened the pull request:
Sorry for this second request in 2 days (see #1663), but I didn't know we could also use Netlify.

It would be better for our application because we need the "redirect" feature of Netlify, to be able to redirect smoothly users from the root level `/` to `/2017/en` for example.

FYI we have already a `CNAME` file on Netlify, filled with: `risingstars.js.org`

Thank you again!

- [x] There is reasonable content on the page (see: [No Content](https://github.com/js-org/dns.js.org/wiki/No-Content))
- [x] I have read and accepted the [ToS](http://dns.js.org/terms.html)

indus commented:
No problem. Support for other providers or selfhosting is just not promoted and limited to "lets give it a try".
I've made the changes in JS.ORGs zonefile but all I'm getting from netlify is a "Not found". Looks like everything is working on my side but netlify is not picking up the incoming request. Can you double-check the CNAME and maybe remove and re-add?

indus commented:
From here: https://www.netlify.com/docs/custom-domains/

> After you add a custom domain to a site, the domain name will normally appear grayed out in the Custom domains panel, with a Check DNS configuration link next to it. This means Netlify cannot serve your site correctly under that domain.

michaelrambeau commented:
Thank you very much Stefan, I have removed and re-added the `CNAME` file, and it seems to work now.

In addition, in the Netlify settings, I added `risingstars.js.org` as a "subdomain" (see screenshot below) but I'm not sure it was needed (if you know about that, let me know, so that I can clean up things!).

![image](https://user-images.githubusercontent.com/5546996/34648053-95386de4-f3d5-11e7-9d0d-ac33672203d4.png)
`

indus commented:
Always wanted but never had the time to take a look at netlify for one of my own projects. So I really have no idea what the option is good for. Nevertheless welcome back https://risingstars.js.org

michaelrambeau commented:
Hello Stefan @indus 
Thank you very much for everything.
It's very convenient to be able to deploy to Netlify just by pushing to GitHub.

The site https://risingstars.js.org/ was launched successfully this week.
So far I'd say it's not as popular as last year, when it was hot on HackerNews.

indus commented:
@michaelrambeau You are probably using some analytics software on you side but nevertheless I wanted to show you what your page contributed to the number of requests on JS.ORG-Pages in the last days (as measured by Cloudflare). The pink line is what I would guess the graph would have looked like without your page. At least you shortend the normal the 2-day weekend vally to a single day ;-)
![zeichenflache 1](https://user-images.githubusercontent.com/2046274/35250638-e00cb7b8-ffd7-11e7-9ef4-a2a10ea9aa5c.png)
(Holiday season and first week in 2018 was a bit lower than normal).
